target/linux/ipq807x/base-files/etc/board.d/01_leds

```diff
@@ -21,6 +21,15 @@ qnap,301w)
 	ucidef_set_led_netdev "10G_1" "10G_1" "green:10g_1" "eth4"
 	ucidef_set_led_netdev "10G_2" "10G_2" "green:10g_2" "eth5"
 	;;
+netgear,rax120)
+	ucidef_set_led_default "power" "POWER" "white:power" "1"
+	ucidef_set_led_default "wps" "WPS" "white:wps" "1"
+	ucidef_set_led_netdev "wan" "WAN" "white:wan" "eth4"
+	ucidef_set_led_wlan "wlan2g" "WLAN2G" "white:wlan2g" "wlan1"
+	ucidef_set_led_wlan "wlan5g" "WLAN5G" "white:wlan5g" "wlan0"
+	ucidef_set_led_usbport "usb" "USB" "white:usb1" "usb2-port2" 
+	ucidef_set_led_usbport "usb" "USB" "white:usb2" "usb4-port2"
```

Comment:
USB LED-s can be set via DTS

Reply:
Thanks! Corrections pending.

Reply:
I was hoping to do that at least but I was having trouble here because the ipq8074.dtsi does not include a trigger-source for the USB ports.

Reply:
You have to add it in the device DTS

Reply:
I guess I'm just sure where to place the trigger-source's to accomplish that? USB under is 807x is not as clear as it is on other devices.

Reply:
Check the NBG6617 and others in IPQ40xx, its the exact same logic.

Reply:
Thanks! I think I'm following now. 

We define ports under dwc and the ports act as the trigger source. I'll give this a try.